BookPlayer/Library/BaseListViewController.swift

```diff
@@ -213,21 +214,20 @@ extension BaseListViewController: UITableViewDataSource {
         cell.artwork = item.artwork
         cell.title = item.title
         cell.isPlaylist = item is Playlist
-        cell.playbackState = .Stopped
-        cell.titleColor = UIColor.black
+        cell.playbackState = .stopped
 
         if let book = item as? Book {
             cell.subtitle = book.author
 
-            cell.progress = item.percentCompleted / 100.0
+            cell.progress = item.currentTime / item.duration
```

Comment:
you could use the introduced `progress` calculated var in the `Book` class

Reply:
Didn't have the property when I started, but I should definitely do that :D